src/librustc_resolve/lib.rs

```diff
@@ -1584,6 +1588,7 @@ impl<'a> Resolver<'a> {
                     TypeNS => Scope::BuiltinTypes,
                     ValueNS => break, // nowhere else to search
                     MacroNS => Scope::BuiltinAttrs,
+                    LifetimeNS => bug!("shouldn't be resolving things in lifetime ns"),
```

Comment:
`'static` can't be shadowed so likely it wouldn't need a `BuiltinLifetimes`, meaning `LifetimeNS` can be like `ValueNS`.